src/test/java/com/example/Bloggy/BlogPostControllerTest.java

```diff
@@ -64,6 +66,19 @@ void testGetBlogPostById() throws Exception {
     }
 
 
+    @Test
+    @DisplayName("GET /posts/1 - Not found")
+    void testGetBlogPostByIdNotFound() throws Exception {
```

Comment:
Imo better name would be: GetBlogPostByIdShouldReturnNotFoundIfNoPostFound() test names should best reflect their functions, in complex applications tests are counted in thousands, we want to easily determine what does not work simply by looking on name of failed test